### ISSUE TITLE
Bug(?) Using recordfinder`s popup in the popup-form of Relation behavior

### ISSUE THREAD

**Issue by NemoD503** [octobercms/october#1016]
I try to use recordfinder in the form of Relation behavior and I get an exception.
My **relation behavior config** file contains the link to form config (fields.yaml)

     items:
         label: ....
         ...
         form: ~/plugins/draft/catalog/models/combinationitem/fields.yaml
     ..........

In that **fields.yaml** I have one field with type recordfinder

    fields:
        product:
            type: recordfinder
        ........

So, when I try to use search or sorting columns in the recordfinder`s popup, I get an exception, something like that ..this widget was not binded to the controller..

The question is, is it bug, or it is normal behavior. Is it possible to use popup in the popup?
One picture is better, than text

![clip](https://cloud.githubusercontent.com/assets/4633952/6766555/b4601374-d01b-11e4-9f6f-a4e8421f0b28.png)

P.S. sorry for my English

**Comment by daftspunk** [comment]
It should be possible, can you tell me what Build you are using? This was recently fixed.

**Comment by NemoD503** [comment]
It was Build 230, now I use 234, and I still get this error.

**Comment by vbarinov** [comment]
Hey. Got something related. I'm using `dependsOn` in popup form (Build 226)


![image](https://cloud.githubusercontent.com/assets/1442435/6906966/b2edad12-d74b-11e4-8a2e-88f45490ebd9.png)

**Comment by daftspunk** [comment]
This should now be fixed as an outcome of #1062

**Comment by slavixdnb** [comment]
I get pretty similar exception using **recordfinder** in create popup form. (Build 317)
Steps to reproduce bug in October Test Plugin:

1. In **plugins/october/test/models/comment/fields.yaml** change **post** field type to **recordfinder** and specify **list** parameter like this
```
post:
        type: recordfinder
        nameFrom: name
        list: $/october/test/models/post/columns.yaml
```
![recordfinder-exception](https://cloud.githubusercontent.com/assets/644543/14047764/7b27d550-f2b2-11e5-9bf6-6fe4fb6ad961.png)
